vsprojects/common.vcxproj.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <ForcedIncludeFiles>config.h</ForcedIncludeFiles>
```

Comment:
is that required ? What does it do ?

Reply:
This is equivalent to `/FI` on the command line, which force includes the header file before any other header files.